openraft/src/lib.rs

```diff
@@ -1,6 +1,4 @@
 #![doc = include_str!("../../README.md")]
-#![cfg_attr(feature = "bt", feature(backtrace))]
-#![cfg_attr(feature = "bench", feature(test))]
```

Comment:
`feature(test)` is still required when `bench` is enabled.

It's my fault, the benchmark is not covered in CI.